protocol/statetracker/pairing_updater.go

```diff
@@ -85,6 +90,18 @@ func (pu *PairingUpdater) Update(latestBlock int64) {
 	pu.nextBlockForUpdate = nextBlockForUpdateMin
 }
 
```

Comment:
can we add all new functions next to each other and maintain hierarchy? for internal functions caller above calle